packages/flutter_tools/lib/src/project.dart

```diff
@@ -881,6 +881,40 @@ $javaGradleCompatUrl
     }
     return AndroidEmbeddingVersionResult(AndroidEmbeddingVersion.v1, 'No `<meta-data android:name="flutterEmbedding" android:value="2"/>` in ${appManifestFile.absolute.path}');
   }
+
+  static const bool _impellerEnabledByDefault = false;
```

Comment:
This should have a TODO and linked issue to switch it over when we change the default.

Reply:
Done.